helm-chart/images/binderhub/requirements.in

```diff
@@ -4,3 +4,7 @@
 #    ./dependencies freeze --upgrade
 #
 google-cloud-logging
+
+# pins for our image
+kubernetes==9.*
+jupyterhub==1.1.*
```

Comment:
```suggestion
# pins for our image
# kubernetes is pinned to 9 because we have observed a performance issue in 12.
kubernetes==9.*
# jupyterhub is pinned to match the JupyterHub Helm chart's version of jupyterhub
jupyterhub==1.1.*
```